README.md

```diff
@@ -1,2 +1,33 @@
-# Ner4Opt
-Ner4Opt: Named-Entity Recognition for Optimization
+# Named Entity Recognition for Optimization (Ner4Opt) Library
+
+Given an optimization problem in natural language, the library extracts **six named entities**:
+
+- **CONST_DIR** (constraint direction)
+- **LIMIT** (limit)
+- **OBJ_DIR** (objective direction) 
+- **OBJ_NAME** (objective name)
+- **PARAM** (parameter)
+- **VAR** (variable)
+
+## Installation
+
+```bash
+pip install . 
+```
+
+## Quick Start
+
+```python
```

Comment:
This usage example IS the heart of all the library here... and it is quire undersold as is.. You need to polish this more.

# Import the Ner4Opt Library
from ner4opt import Ner4Opt

# Problem Description (I am changing example with problem_description, do the same in other places) 
problem_description = "Cautious Asset Investment has a total of $150,000 to manage and decides to invest it in money market fund, which yields a 2% return as well as in foreign bonds, which gives and average rate of return of 10.2%. Internal policies require PAI to diversify the asset allocation so that the minimum investment in money market fund is 40 % of the total investment. Due to the risk of default of foreign countries, no more than 40% of the total investment should be allocated to foreign bonds. How much should the Cautious Asset Investment allocate in each asset so as to maximize its average return?"

# Ner4Opt Model options: lexical, lexical_plus, <add the rest> ..
ner4opt = Ner4Opt(model="hybrid") 

# Extract the list of dictionaries corresponding to each word in the problem description. (this is for each word, if I remember correctly??)
# Each dictionary holds keys for start, end, <add the rest>.. 
entities = ner4opt.get_entities(problem_description)

# Output
print(entities) # if it is better, replace this with how to pretty print it (may be json.print() or prettyprint(), sth like that) 

# Here add the output here and comment out .. if too large output, select a few to give the user a sense of output
# ..
# ..

Reply:
# BTW, in the problem description, there are whitespaces before/after single characters (like , % ? etc..) 

I fixed that in your slides as well. I am fixing it in the above too. IS this gonna be a problem? I hope not? Please also fix in other places

Reply:
The reason for this is, the training, dev, test data are all spacy tokenized. The code works even when the example is not spacy tokenized, but it is better to make sure it is. I added a line about this in data characteristics. It is hard to check for this, we can caution people to usage spacy tokenized example problems as our models are trained on such text. I did mention this in data characteristics, I will also add it at other places, wherever you might think necessary.